sinonjs/sinon
# Fix broken link to matchers in spies and spy-call docs

#### Purpose (TL;DR) - mandatory


- fixes relative links for `/matchers` in `/spies` and `/spy-call` documentation pages
- affected versions: `v6.1.4` to `latest`

##### Screenshots of Repro:

![image](https://user-images.githubusercontent.com/1915925/59231800-02439800-8bb0-11e9-9fc3-f1c65279c9cb.png)

![image](https://user-images.githubusercontent.com/1915925/59231823-18515880-8bb0-11e9-8379-81ad182b3f12.png)

![image](https://user-images.githubusercontent.com/1915925/59231882-37e88100-8bb0-11e9-9c41-8348d964767d.png)








 #### How to verify - mandatory

1. Check out this branch
2. `npm install`
3. Follow the steps to [serve the documentation site](https://github.com/sinonjs/sinon/blob/master/docs/CONTRIBUTING.md#running-the-documentation-site-locally)
4. Go to http://localhost:4000/releases/v7.3.2/spies/
5. Find `(see matchers)` on the page; click the link; verify it leads to the matchers page
6. Go to http://localhost:4000/releases/v7.3.2/spy-call/
7. Find `(see matchers)` on the page; click the link; verify it leads to the matchers page

 #### Checklist for author
